clients/src/main/resources/common/message/ShareFetchResponse.json

```diff
@@ -39,6 +39,8 @@
       "about": "The top-level response error code." },
     { "name": "ErrorMessage", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
       "about": "The top-level error message, or null if there was no error." },
+    { "name": "AcquisitionLockTimeout", "type": "int64", "versions": "0+",
```

Comment:
Please can you add `Ms` to the field name? In RPCs, the convention for timeouts is always to have `Ms` on the end, such as `RebalanceTimeoutMs`. Also, they are always `int32`, which is probably appropriate here too.

Reply:
Done.